lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -3991,7 +3975,7 @@ void PatchInOutImportExport::storeValueToGsVsRing(Value *storeValue, unsigned lo
             ConstantInt::get(Type::getInt32Ty(*m_context), coherent.u32All) // glc, slc, swz
         };
         emitCall("llvm.amdgcn.raw.tbuffer.store.i32", Type::getVoidTy(*m_context), args, {}, insertPos);
-      } else if (m_gfxIp.major == 10) {
+      } else {
```

Comment:
This looks like an unrelated change.

Reply:
The original code is as below:
```
else if (m_gfxip.major == 10) else {
        CoherentFlag coherent = {};
        coherent.bits.glc = true;
        coherent.bits.slc = true;
        coherent.bits.swz = true;
        Value *args[] = {
            storeValue,                                                          // vdata
            m_pipelineSysValues.get(m_entryPoint)->getGsVsRingBufDesc(streamId), // rsrc
            ringOffset,                                                          // voffset
            gsVsOffset,                                                          // soffset
            ConstantInt::get(Type::getInt32Ty(*m_context), BUF_FORMAT_32_UINT),  // format
            ConstantInt::get(Type::getInt32Ty(*m_context), coherent.u32All)      // glc, slc, swz
        }; 
```
I think the combined buffer format is used for Gfx10+, is right?

Reply:
Yes the combined buffer format is used for GFX11.